ami/ml/migrations/0030_remove_custom_api_key_fields.py

```
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.10 on 2026-03-27 20:50
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("ml", "0029_processingserviceapikey"),
+    ]
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)